static_analyzer/incremental_orchestrator.py

```
@@ -214,14 +214,16 @@ def _add_outbound_edges_from_changed_files(
         call_sites = source_inspector.find_call_sites(file_path)
         if not call_sites:
             continue
-        queries = [(file_path, line, char) for line, char in call_sites]
+        queries = [(file_path, site.line - 1, site.column - 1) for site in call_sites]
         try:
             definition_results, _ = engine_client.send_definition_batch(queries)
         except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)